### ISSUE TITLE
Question: Why was Vue Native deprecated?

### ISSUE THREAD

**Issue by chuanhaisoft** [GeekyAnts/vue-native-core#354]
Why was Vue Native deprecated?

**Comment by cwilby** [comment]
Given the absence of an explanation, I speculate that Geeky Ants has decided to stop funding open source projects.

**Comment by cwilby** [comment]
Did those geeky little ants close source a thing? 👀

Who knows!

**Comment by rijulg** [comment]
Could only find this while looking around the project for the same question

https://github.com/GeekyAnts/vue-native-core/blob/30e6e65e2e61eb6ec1d3075d868430d24f43f5b8/src/core/instance/index.js#L24

**Comment by cwilby** [comment]
🤌💰

**Comment by SwenVanZanten** [comment]
> Why was Vue Native deprecated?

Probably because the maintainers migrated away from this project and moved to well maintained alternatives. Remember that any open source project can be "abandoned" at any given moment. On the other you yourself can just fork the project and give it new life 😉

**Comment by SwenVanZanten** [comment]
> @SwenVanZanten Cool way to shit on [the contributors](https://github.com/GeekyAnts/vue-native-core/graphs/contributors) of this project 🤦‍♂️

You probably miss understood my Dutch directness, my intention was not to shit on the work of the contributors. Thing is, maintaining a framework must have a purpose for a business. Otherwise it becomes a costly reliability. But anyway I'm just guessing here.

**Comment by ttodua** [comment]
of course, no one has a right to say something negative on contributors, but one thing has to be mentioned - the worst and unhelpful way to your audience is to deprecate the project with just message: "project is deprecated" without mentioning any possible alternatives.

**Comment by Vampire-js** [comment]
But well, its really sad it's deprecated. For those looking for other options, Ionic Vue and Nativescript Vue are really good

**Comment by vallemar** [comment]
Use Nativescript vue for native apps!

**Comment by nxmndr** [comment]
> without mentioning any possible alternatives.

and the WHY.

**Comment by fatihtkale** [comment]
Dont use nativescript-vue, its horrible, DX is so bad, it always crashes, comes with weird errors, and all in all not a good development tool, i would adivce using ionic vue much better and actually gives you a good experience
